main.nf

```diff
@@ -86,8 +92,120 @@ workflow {
         params.outdir,
         params.monochrome_logs,
         params.hook_url,
-        multiqc_report
+        MULTIQC.out.report.toList()
     )
+
+    publish:
+    multiqc_data >> 'multiqc'
+    multiqc_plots >> 'multiqc'
+    multiqc_report >> 'multiqc'
+}
+
+output {
+    'bowtie1' {
+        path 'bowtie1'
+        mode 'copy'
```

Comment:
Might be easier to set the mode in the config:
```groovy
workflow.output.mode = 'copy'
```

Then you can override for specific targets as needed